packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: LGPL-3.0-only
-pragma solidity 0.8.19;
+pragma solidity 0.8.22;
```

Comment:
Ah this was just so that I could import it in GenArt721Generator. Do you think I should break out an interface instead?

Reply:
oh yeah, I would probably just import IDependencyRegistryV0 in the generator contract, since the interface shouldn't be version locked. any functions that are needed can just be added to the interface.

Reply:
resolved in [2254aea](https://github.com/ArtBlocks/artblocks-contracts/pull/1320/commits/2254aea5bc33ed01e3cd604c076356eec8d92ff0)